src/v2/ion/i18nTransformer.js

```diff
@@ -45,11 +45,17 @@ const I18N_OVERRIDE_MAPPINGS = {
   'identify.credentials.passcode': 'primaryauth.password.placeholder',
   'identify.rememberMe': 'remember',
 
-  'select-authenticator-enroll.authenticator.email': 'oie.email',
-  'select-authenticator-enroll.authenticator.password': 'oie.password',
-  'select-authenticator-enroll.authenticator.phone': 'oie.phone',
+  'select-authenticator-enroll.authenticator.email': 'oie.authenticator.email.label',
```

Comment:
Made these labels more specific for future. Also removed labels to accommodate what comes from response